.github/workflows/reusable-build-test-release.yml

```diff
@@ -615,9 +615,27 @@ jobs:
           version: ${{ steps.BuildVersion.outputs.VERSION }}
       - name: Slim Package
         id: slim
-        uses: splunk/addonfactory-packaging-toolkit-action@v1
-        with:
-          source: ${{ steps.uccgen.outputs.OUTPUT }}
+        run: |
+          pip install semantic_version==2.6.0
```

Comment:
I believe we don't need to have this as a separate `pip install`, it should be covered in `splunk-packaging-toolkit`.